<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire de garanties Hermès</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icon-hermes.png</normaloff>:/images/icon-hermes.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="listeProduits"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>25</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveauProduit"/>
    <addaction name="actionSupprimerProduit"/>
    <addaction name="separator"/>
    <addaction name="actionGererMagasins"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAide"/>
    <addaction name="actionAPropos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNouveauProduit"/>
   <addaction name="actionSupprimerProduit"/>
   <addaction name="separator"/>
   <addaction name="actionGererMagasins"/>
  </widget>
  <action name="actionNouveauProduit">
   <property name="text">
    <string>Nouveau produit...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSupprimerProduit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Supprimer le produit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionGererMagasins">
   <property name="text">
    <string>Gérer les magasins...</string>
   </property>
  </action>
  <action name="actionAide">
   <property name="text">
    <string>Centre d'aide</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAPropos">
   <property name="text">
    <string>À propos...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
